src/mount/writedata.cc

```diff
@@ -213,6 +213,7 @@ static std::condition_variable fcbcond;
 static uint32_t fcbwaiting = 0;
 static int64_t freecacheblocks;
 static inodedata **idhash;
+static bool ignoreflush = 0;
```

Comment:
Consider to set this to false as it is done in some other static bool variables around the repo.